app/policies/mfa_policy.rb

```diff
@@ -22,10 +22,7 @@ def more_than_two_factors_enabled?
   end
 
   def sufficient_factors_enabled?
-    mfa_user.enabled_mfa_methods_count > 1 ||
-      (FeatureManagement.backup_codes_as_only_2fa? &&
-      mfa_user.backup_code_configurations.to_a.length.positive? &&
-      !signup_flag)
+    mfa_user.enabled_mfa_methods_count > 1 || mfa_user.backup_code_configurations.to_a.length.positive?
```

Comment:
Good cleanup. Looks like we can also pull the `signup` param from the initializer and drop the getter for `signup_flag`

Reply:
Good idea.  Done.